server/protocols/jmap-draft-integration-testing/rabbitmq-jmap-draft-integration-testing/pom.xml

```diff
@@ -117,6 +117,12 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-guice-mailbox</artifactId>
```

Comment:
These imports are evil IMO

Reply:
I was hesitating between this and duplicate every the retry backoff binding statement every where. I choose the less evil.

Reply:
reverted